miditok/tokenizations/mmm.py

```diff
@@ -314,6 +311,7 @@ def tokens_to_midi(
                     and den != current_time_signature.denominator
                 ):
                     time_signature_changes.append(TimeSignature(num, den, current_tick))
+                    # ticks_per_bar = self._compute_ticks_per_bar(time_signature_changes[-1], time_division)
```

Comment:
I left this on purpose, because line 294 with `current_bar = current_tick // ticks_per_bar` will also need to be changed, because if another time signature appears, we need to consider different `ticks_per_bar` as in Octuple.

Reply:
It is the question of whether I should implement full time signature support for MMM now or let it work with at least 4/4 or a single time signature in MIDI.

Reply:
Ok I just saw that MMM didn't really decoded the TS values (only added them to the list of TS). I might have been lazy when implementing it, I'll handle it later